performance_test/helper_scripts/regression_checkers/ApexComparison.py

```diff
@@ -0,0 +1,554 @@
+# Copyright 2017 Apex.AI, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Provives tools to compare 2 apex performance log files."""
+import csv
+from datetime import datetime
+import itertools
+import logging
+from os import listdir
+from os.path import isfile
+from os.path import join
+import time
+
+
+class ApexComparision():
+    """Class to perform a comparison between 2 test log files."""
+
```

Comment:
This class felt a bit Java-ish, not bad per se, but not very Pythonic. In Python it's uncommon to have a class that only exposes one public method, it's far more common to just use free functions unless you need to store state.